Changes.md

```diff
@@ -27,6 +27,7 @@ Fixes
 - Arnold :
   - Fixed translation of `UsdPreviewSurface` normal maps.
   - Fixed translation of `UsdPreviewSurface` `specularColor` fallback value.
+- GraphEditor : Fixed error caused by `Alt+E` keypress on panels other than the Viewer, HierarchyView or NodeEditor.
```

Comment:
The LightEditor also still responds to `Alt+E`, which I think is intended?

Reply:
Indeed - well spotted. I've fixed that up, and am merging. Thanks!